docs/src/configuration/routes/https.md

```diff
@@ -175,12 +175,21 @@ This breakdown leaves you with 21-23 characters to work with naming your branche
 
 To be able to provide a valid SSL-certificate, Let's Encrypt needs to make sure that the requester is entitled to receive the SSL-certificate it asked for (usually through the presence of a specific token on the DNS zone of that domain).
 
-This ownership verification is achieved through the so called _Challenge_ step.
+This ownership verification is achieved through the so called _Challenge_ step, more background information can be found in the [Let's Encrypt Documentation](https://letsencrypt.org/docs/challenge-types/).
 
 Sometimes, that verification fails which will result in the following error-message:
 `Couldn't complete challenge [HTTP01: pending | DNS01: pending | TLSALPN01: pending]`
 
-Usually this issue can then be resolved by [redeploying the impacted environment](https://docs.platform.sh/development/troubleshoot.html#force-a-redeploy).
+For the DNS challenge to work, the domains and subdomains should point directly to your platform.sh's cluster (unless using a CDN).
+```
+  E: Error validating domain www.some-example.platform.sh: Couldn't complete challenge [HTTP01: pending | DNS01: pending | TLSALPN01: pending]
+  Unable to validate domains www.some-example.platform.sh, will retry in the background.
+```
+In that example the `www.some-example.platform.sh` subdowmain does not point to the cluster.
+The step-by-step DNS setup can be found [here](/domains/steps.html).
```

Comment:
```suggestion
The step-by-step DNS setup can be found [here](/domains/steps/_index.md).
```

Reply:
@trolologuy for future reference on these

We like to have internal links be relative, rather than absolute like you have elsewhere. I've set up Hugo to run checks that make the build fail completely if that page has been moved or renamed, so we can minimize broken links. 

For this edit, a note on how that works. All relative links:
- point to the *file* in Hugo, rather than the final built destination
- are checked to see if that *file* exists

You see that list pages (sections) like this one are nested. So `domains/steps.html` is created from a subdirectory with children, and that actual final page comes from `domains/steps/_index.md`.